authentication/example-providers/psk/src/main/java/io/deephaven/authentication/psk/PskAuthenticationHandler.java

```diff
@@ -69,7 +70,7 @@ public void initialize(String targetUrl) {
         logger.warn().append("================================================================================").endl();
```

Comment:
No, the logger api doesn't offer that - this would need to be a "write this directly to the console", or try to stick ascii control sequences in here and hope they show up in the output. Note that the logger itself will end up on std out with ascii colors (if supported by whatever the logger is being written to):
![screenshot848](https://github.com/deephaven/deephaven-core/assets/241630/7158f6ee-ceee-49cc-a9e3-fa7676cc5ccc)